extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -98,16 +130,19 @@ const Track = memo( ( { track, isActive, isPlaying, isError, selectTrack, index
 } );
 
 const Playlist = memo( ( { tracks, selectTrack, currentTrack, playerState } ) => {
+	const trackRefs = [];
```

Comment:
Is there a better way to store these `trackRefs`? If I put it outside this function, like this:
```
const trackRefs = [];

const Playlist = memo( ( { tracks, selectTrack, currentTrack, playerState } ) => {
...
```

then it gets used for all Playlists. So, if there are multiple playlists, they will use the same `trackRefs` array, which is not good. I need `trackRefs` to be scoped to just the Playlist.